charts/console/Chart.yaml

```
@@ -30,7 +30,7 @@ type: application
 version: 0.7.0
```

Could you please bump the version also. 
```suggestion
version: 0.7.1
```